WalletWasabi/Blockchain/Keys/KeyManager.cs

```diff
@@ -116,68 +125,48 @@ public WpkhDescriptors GetOutputDescriptors(string password, Network network)
 		return WpkhOutputDescriptorHelper.GetOutputDescriptors(network, MasterFingerprint.Value, GetMasterExtKey(password), AccountKeyPath);
 	}
 
-	[JsonProperty(Order = 1)]
-	[JsonConverter(typeof(BitcoinEncryptedSecretNoECJsonConverter))]
 	public BitcoinEncryptedSecretNoEC EncryptedSecret { get; }
 
-	[JsonProperty(Order = 2)]
-	[JsonConverter(typeof(ByteArrayJsonConverter))]
 	public byte[] ChainCode { get; }
 
-	[JsonProperty(Order = 3)]
-	[JsonConverter(typeof(HDFingerprintJsonConverter))]
 	public HDFingerprint? MasterFingerprint { get; private set; }
 
-	[JsonProperty(Order = 4)]
-	[JsonConverter(typeof(ExtPubKeyJsonConverter))]
 	public ExtPubKey ExtPubKey { get; }
 
-	[JsonProperty(Order = 5)] public bool SkipSynchronization { get; private set; } = false;
+	public bool SkipSynchronization { get; private set; } = false;
 
-	[JsonProperty(Order = 6)]
 	public int MinGapLimit { get; private set; }
 
-	[JsonProperty(Order = 7)]
-	[JsonConverter(typeof(KeyPathJsonConverter))]
 	public KeyPath AccountKeyPath { get; private set; }
 
+	[JsonIgnore]
```

Comment:
Personally, I prefer to ADD property with an attribute instead of ignoring it. In case of any code change, the property can easily get serialized unintentionally.

Reply:
Yes, but this is not the final code but the minimal and safest path of less resistance that I found. The code needs to chnage and the the real problem is that the KeyManager persists itself (the Config classes have the same problem btw) instead of perisiting a data object.